src/Game.java

```diff
@@ -0,0 +1,91 @@
+import java.io.FileNotFoundException;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Scanner;
+
+/**
+ * Created by EddyJ on 6/21/16.
+ */
+public class Game {
+    private Player player1;
+    private Player player2;
+    private int sticks;
+    private Player currentPlayer = null;
+    String loser = null;
+    String winner = null;
+    private Scanner scanner = new Scanner(System.in);
+
+    public Game(Player player1, Player player2){
+        this.player1 = currentPlayer = player1;
+        this.player2 = player2;
+    }
+
+    public int getSticks() {
+        return sticks;
+    }
+
+    public String getLoser() {
+        return loser;
+    }
+
+    public String getWinner() {
+        return winner;
+    }
+
+    public String setLoser(Player name) {
+        return loser = name.getName();
+    }
+
+    public String setWinner(Player name) {
+        return winner = name.getName();
+    }
+
+    public int stickTotal(){
+        sticks = 0;
+        System.out.println("Let's play Game of Sticks!!!!!!");
+        do {
+            System.out.println("How many sticks would you like to begin with? (10-100)");
+            sticks = Integer.parseInt(scanner.nextLine());
+        if(sticks >= 10 && sticks <= 100){
```

Comment:
This is a condition the while loop is already testing.  If you want to put something in there then the logic should be outside of the loop as this is redundant.